plugins/plugin-webpack/plugin.js

```diff
@@ -361,10 +363,11 @@ module.exports = function plugin(config, args = {}) {
             reject(err);
             return;
           }
-          if (stats.hasErrors()) {
+          if (stats.hasErrors() || (stats.hasWarnings() && !ignoreWarnings)) {
```

Comment:
it would be great if these could still be output, even if `ignoreWarnings` is `true`. How about something like:

```js
const info = stats.toJson(extendedConfig.stats);
if (stats.hasErrors()) {
  console.error('webpack error:\n' + info.errors.join('\n'));
  reject(Error(`webpack failed with ${info.errors} error(s).`));
}
if (stats. hasWarnings()) {
  console.error('webpack warning:\n' + info.errors.join('\n'));
  if (!ignoreWarnings) {
    reject(Error(`webpack failed with ${info.warnings} warnings(s).`));
  }
}
```

```